tests/Unit/PointwiseFunctions/AnalyticSolutions/Poisson/Test_Lorentzian.cpp

```diff
@@ -70,5 +85,13 @@ SPECTRE_TEST_CASE(
   pypp::SetupLocalPythonEnvironment local_python_env{
       "PointwiseFunctions/AnalyticSolutions/Poisson"};
   // 1D and 2D solutions are not implemented yet.
-  test_solution<3>();
+  using AffineMap = domain::CoordinateMaps::Affine;
+  using AffineMap3D =
+      domain::CoordinateMaps::ProductOf3Maps<AffineMap, AffineMap, AffineMap>;
+  test_solution(
+      Mesh<3>{12, Spectral::Basis::Legendre,
+              Spectral::Quadrature::GaussLobatto},
+      domain::CoordinateMap<Frame::Logical, Frame::Inertial, AffineMap3D>{
+          {{-1., 1., -0.5, 0.5}, {-1., 1., -0.5, 0.5}, {-1., 1., -0.5, 0.5}}},
+      1.e-1);
```

Comment:
For my own understanding: what criterion are you using for choosing different tolerances for different solutions and different dimensions?

Reply:
Your comment made me go back and do this more thoroughly. In the _"fixup tolerances for smooth and non-smooth functions"_ commit I add tests that the error is not only within a given tolerance but also _decreases_ as expected. For smooth functions we expect exponential convergence, and for non-smooth functions power-law convergence when increasing the polynomial order. The parameters of the convergence (e.g. the offset and scaling of the exponential) depend on the problem and the domain, so I determined those experimentally. For example, these are a bunch of norms that show how the Lorentzian solution converges, and an exponential tolerance with C1=5 and C2=1.2:

![Unknown-24](https://user-images.githubusercontent.com/746230/66408649-9d7a3c00-e9ef-11e9-9ec5-c1e9335eea8d.png)

Please let me know if this makes things more clear!